bundle/language_en.properties

```diff
@@ -645,7 +655,7 @@ angal.groupsbrowser.mainmenu.txt
 angal.groupsbrowser.menuitembrowser.title                                                              = Menu Item Browser
 angal.groupsbrowser.newgroup.title                                                                     = New Group
 angal.groupsbrowser.pleaseinsertavalidusergroupname.msg                                                = Please insert a valid user group name.
-angal.groupsbrowser.alreadysoftdeleted.msg                                                             = The group is already set as deleted.
+angal.groupsbrowser.softdeletegroup.fmt.msg                                                            = Could not delete the group '{0}' because it is used somewhere. Do you want to mark it as deleted ?
```

Comment:
```suggestion
angal.groupsbrowser.softdeletegroup.fmt.msg                                                            = Could not delete the group '{0}' because it is used somewhere. Do you want to mark it as deleted?
```